demo/components/http-endpoint/http-endpoint-service.js

```diff
@@ -41,16 +41,18 @@ app.service('HttpEndpoint', function($q, $http, $timeout, GENERAL_CONFIG) {
     load : function(resource, graphUri) {
       var deferred = $q.defer();
       var cacheLocation = endpoint;
-      if (graphUri) {
-        cacheLocation = cacheLocation + '?uri='+resource+'&context-uri=' + graphUri;
-      } else {
-        cacheLocation = cacheLocation + '?uri='+resource+'&context-uri=' + resource;
+      var context-uri = resource
+	 if (graphUri) {
+        context-uri = graphUri
       }
       
+	  	
+	
       $http.post(cacheLocation, {
         headers : {
           'Content-Type' : 'application/x-www-form-urlencoded'
-        }
+        },
+		data: { uri: resource, context-uri: graphUri }
```

Comment:
Shouldn't this line start at same level as "headers" line?

Reply:
yes. That's a weird bug.

Reply:
True, maybe some weird last minute typo

Reply:
I missunderstood what you said. What do you mean?

Reply:
I mean the indentation in line 55 is wrong. Both the lines 52 and 55 should have same indentation right?

Reply:
Ah, yes.. I also fixed that.